packages/components/src/interfaces/complex/Hero.ts

```diff
@@ -84,4 +86,7 @@ export const HeroSchema = Type.Composite(
 )
 
 export type HeroBackgroundImageProps = Static<typeof HeroBackgroundImageSchema>
-export type HeroProps = Static<typeof HeroSchema>
+export type HeroProps = Static<typeof HeroSchema> & {
+  site: IsomerSiteProps
+  LinkComponent?: any // Next.js link
```

Comment:
Wah I feel like I keep kicking this can down the road, I can standardise them to use a `LinkComponentType` but for now I will put it as `any` first. Initial glance is that it is [not very straightforward to type the Next.js link](https://github.com/vercel/next.js/blob/fcde8fc1a52857e0cc72ee2c385f0105044a4115/packages/next/src/client/link.tsx#L35) without directly importing it

Reply:
Centralised the definition in ffbf62e97d0b6f2d15095c6daa3a61df28ff1302.